packages/oui-select-picker/src/select-picker.html

```diff
@@ -26,7 +26,9 @@
         </span>
 
         <!-- Label -->
-        <span class="oui-select-picker__label">
+        <span class="oui-select-picker__label"
+              id="{{:: $ctrl.id }}Label"
+              ng-transclude="labelSlot">
             <span id="{{:: $ctrl.id }}Label"
```

Comment:
Careful, you'll get a double ID here. You could remove the one on this `span` ;)